client/src/components/Temperature.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Card }  from 'react-bootstrap';
+import "../css/themePalette.css";
+import "../css/Temperature.css";
+
+class Temperature extends Component {
+  render () {
+    return (
+      <Card>
+        <Card.Body>
+            <Card.Title id="Temperature-temp">
+              98.8F
+            </Card.Title>
+            <Card.Subtitle id="Temperature-avgTempText">
+              Average Temp
+            </Card.Subtitle>
+        </Card.Body>
+      </Card>
```

Comment:
Unless I'm missing something, do we necessarily need a `Card` here? I like the fact that the temperature and text "Average Temp" are tightly coupled like this, but I'm wondering if there's a way to get rid of the border? I feel that in the actual UI, that could look a little out of place.

Reply:
I can do that-- I wanted to use as much react-bootstrap as i could before i made it a div, but i could switch it over to a div. 

I did notice the rounded border for the card. I wonder-- and this is going much further into the design-- perhaps even a shadow could be added to the card border itself to give it some depth.

Reply:
I'll defer that question to @BrianHHough as he's been on top of the design aspect of this, but yeah I'd say either removing the border entirely or adding a showing would be the two routes here 👍

Reply:
@SomeMoosery Hey I removed the border and border radius!

Reply:
Great!